content/devices/crownlte.md

```diff
@@ -0,0 +1,101 @@
+---
+draft: false
```

Comment:
Please add a template file instead, Please refer readme file for updated instructions.